exonum/src/blockchain/schema.rs

```diff
@@ -122,16 +121,16 @@ impl<T: Access> Schema<T> {
     /// This method can be used to build a proof that execution of a certain transaction
     /// ended up with a particular status.
     /// For an execution that resulted in an error, this will be an usual proof of existence.
-    /// If transaction was executed successfully, such a proof will be a proof of absence.
+    /// If the transaction was executed successfully, such a proof will be a proof of absence.
     /// Since the number of transactions in a block is mentioned in the block header, the user
     /// will be able to distinguish absence of error (meaning successful execution) from
-    /// absence of transaction with such an index: if index is less than amount of transactions
-    /// in block, proof denotes successful execution, otherwise transaction with such an index
-    /// does not exist in the block.
+    /// the absence of a transaction with such an index. Indeed, if the index is less
+    /// than amount of transactions in block, the proof denotes successful execution;
+    /// otherwise, the transaction with the given index does not exist in the block.
     ///
-    /// Similarly, execution errors of the `before_transactions` / `after_transactions` hooks can be proven
-    /// to external clients. Discerning successful execution from a non-existing service requires prior knowledge
-    /// though.
+    /// Similarly, execution errors of the `before_transactions` / `after_transactions`
+    /// hooks can be proven to external clients. Discerning successful execution
+    /// from a non-existing service requires prior knowledge though.
```

Comment:
It seems like I screwed up and did not add a note about the required prior knowledge requested by @dmitry-timofeev (see https://github.com/exonum/exonum/pull/1705#discussion_r367431233).

Can you please add it? It can be phrased something like:


"Note that this kind of proof should not be mixed up with a proof of transaction inclusion. To verify that certain transaction was included in the block, consider using a proof for `block_transactions` index."